website/content/docs/connect/cluster-peering/k8s.mdx

```diff
@@ -25,9 +25,26 @@ You must implement the following requirements to create and use cluster peering
 - At least two Kubernetes clusters
 - The installation must be running on Consul on Kubernetes version 0.47.1 or later
 
+### Pre-setup 
+
+1. After provisioning a Kubernetes cluster and setting up your kubeconfig file to manage access to multiple Kubernetes clusters, export the Kubernetes context names for future with `kubectl`. 
+
+  You can discover the context names using either `kubectl config current-context` for the current cluster you are on, or by issuing `kubectl config get-contexts` to get all configured contexts for your kubeconfig file.  
+
+  ```shell-session
+  $ export CLUSTER1_CONTEXT=<CONTEXT for first Kubernetes cluster>
+  $ export CLUSTER2_CONTEXT=<CONTEXT for second Kubernetes cluster>
+  ```
+
 ### Helm chart configuration
 
-To establish cluster peering through Kubernetes, deploy clusters with the following Helm values.
+1. To establish cluster peering through Kubernetes, create a values.yaml file with the following Helm values. 
+   
+  With these values,
+  the servers in each cluster will be exposed over a Kubernetes Load balancer service. This service can be customized
+  using [`server.exposeService`](/docs/k8s/helm#v-server-exposeservice). 
+
+  When generating a peering token from one of the clusters, the address(es) of the load balancer will be used in the peering token, so the peering stream will go through the load balancer in front of the servers. For customizing the addresses used in the peering token, see [`global.peering.tokenGeneration`](/docs/k8s/helm#v-global-peering-tokengeneration).
```

Comment:
```suggestion
  When generating a peering token from one of the clusters, Consul uses the address(es) of the load balancer in the peering token so that the peering stream goes through the load balancer in front of the servers. For customizing the addresses used in the peering token, refer to [`global.peering.tokenGeneration`](/docs/k8s/helm#v-global-peering-tokengeneration).
```
Switched to present tense and active voice per the style guide. Also, do we mean "While generating a peering token . . ." in the first sentence? IF the operation can happen more than once, then we should say "While . . .", but if it is a one-time operation, then "When . . ." is OK.